src/selectors/layers.test.js

```diff
@@ -35,7 +35,11 @@ describe('Selectors', () => {
 
       nodes.forEach(node => {
         // we don't need to check y/height positions if the layer isn't there.
-        if (!node.layer || node.layer === null) {
+        if (
+          !node.layer ||
```

Comment:
If this includes `!node.layer` then `... || node.layer === null || typeof node.layer === 'undefined'` is not necessary? 

It should be only one of the above expressions - I'm inclined to say don't use `!node.layer` here at all, so that `0` is allowable as a layer id (I hope this is the case elsewhere in the code, let me know if not).

Also note that when checking typeof for undefined you must always quote it 
as in `typeof x === 'undefined'`.

Reply:
sure I agree - we can take our the `node.layer === null || typeof node.layer === 'undefined'` cases since `!node.layer` does cover them all